lib/Macros/Sources/SwiftMacros/SwiftifyImportMacro.swift

```diff
@@ -136,6 +156,18 @@ func getPointerMutability(text: String) -> Mutability? {
   }
 }
 
+// Remove std. or std.__1. prefix
+func getUnqualifiedStdName(_ type: String) -> String? {
+  if (type.hasPrefix("std.")) {
+    var ty = type.dropFirst(4)
+    if (ty.hasPrefix("__1.")) {
```

Comment:
Will it always be `__1.`, or could it also be `__2.`?

Reply:
I do not anticipate a change in this, it is the same namespace for the last decade or so. But it is true that we are somewhat closely coupled to this implementation detail.

Reply:
Should we make this adjustment on the C++ side where we can reason about things like inline namespaces?

Reply:
I think it would be reasonable to skip inline namespaces in the Swift names, especially since these names do leak into the diagnostics occasionally. It would be more user friendly. I am not sure if this could ever create name collisions, hopefully not. But I'd address this in a separate PR.

How closely coupled are the compiler and the standard library? In case this code supposed to work with older compiler versions, we'd want to keep the skipping logic here.